app/src/main/java/org/standardnotes/notes/LoginActivity.kt

```diff
@@ -57,7 +57,7 @@ class LoginActivity : AppCompatActivity() {
                 hideProgress()
             }
 
-            override fun onFailure(call: Call<SigninResponse>, t: Throwable) {
+            override fun onFailure(call: Call<SigninResponse>?, t: Throwable?) {
```

Comment:
yeah i mean, I think my recommended change is mostly cosmetic and makes maintenance easier. if the app works, great.